src/timings.rs

```diff
@@ -180,7 +180,10 @@ pub struct NextTimeout {
 }
 
 impl NextTimeout {
-    pub(crate) fn not_zero(&self) -> Option<Duration> {
+    /// Returns the duration of the timeout if the timeout must happen, but avoid null timeouts
```

Comment:
It's not exactly `null` timeout, right? It's 0. Since we have an `Optional` in context, it maybe is a bit misleading?

Reply:
Changed it to "but avoid instant timeouts"

Reply:
Sounds good